semantic/src/site/globals/site.overrides

```diff
@@ -306,23 +325,89 @@ a:hover {
 }
 
 /*--------------------------------
-      General used classes
+      Application Form area
 ---------------------------------*/
 
-// In use for ApplicationHeader around all Application pages
-#application-content {
-  width: 800px;
+.dev-border {
+  // border: 1px solid black;
+  display: inherit;
+}
+
+#application-form {
   display: flex;
   flex-direction: column;
   background-color: @surfacesWhite;
-  padding: 30px 50px 32px;
+  width: @applicationFormWidth;
+  padding: 30px 1rem 32px;
 }
 
-// In use for company title in application header (home/create pages)
-.heading-alt {
-  letter-spacing: @letterSpacingRegular;
-  font-family: @alternativeFont;
-  font-weight: @regularWeight;
+#progress-column {
+  max-width: 205px;
+}
+
+#form-column {
+  padding: 0;
+}
+
+// Seperation between Section title and form elements
+.form-area {
+  margin-top: 30px;
+}
+
+#help-column {
+  max-width: 200px;
+}
+
+/*--------------------------------
+      Application Navigation area
+---------------------------------*/
+
+#app-navigation {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: @footerHeight;
+  box-shadow: @basicBoxShadow;
+  background: @surfacesWhite;
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  box-shadow: @basicBoxShadow;
+  border-top-left-radius: 6px;
+  border-top-right-radius: 6px;
+  z-index: 1000;
+}
+
+#app-navigation-content {
+  display: flex;
+  width: @applicationFormWidth;
+  padding-left: 20px;
+  padding-right: 20px;
+}
+
+#prev-next-links {
+  display: flex;
+  flex-grow: 1;
+  justify-content: space-between;
+  align-items: center;
+}
+
+#prev-next-links p {
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-left: 10px;
+  padding-right: 20px;
+}
+
+/*--------------------------------
+      General used classes
+---------------------------------*/
+
+// In use for ApplicationHeader around all Application pages
+
+.application-form-width {
```

Comment:
Couldn't find where this is used

Reply:
Yeah, good catch. I've replaced this with the variable `@applicationFormWidth`

Reply:
Addressed this and most of the above inline comments: 830e3c13

Reply:
Thanks